.eslintrc.js

```diff
@@ -9,7 +9,12 @@ module.exports = {
     jest: true,
     node: true,
   },
-  globals: ['cy', 'Cypress', 'expect', 'assert'],
+  globals: {
+    cy: 'readonly',
+    Cypress: 'readonly',
+    expect: 'readonly',
+    assert: 'readonly',
+  },
```

Comment:
@tinkertrain by moving these to readonly will they impact cypress tests at all?

cc: @ryan-mulrooney

Reply:
Hi @malikalim, at some point Eslint changed how these globals are declared, so my guess is this rule has not been taking any effect for a while.

What these declarations do is telling Eslint not to tell off the developer when they use these globals in their code (otherwise eslint tells you: "the `cy` variable has not been declared"), `readonly` means that a developer should not reassign their value (by doing something like `cy = 100` in their code), and if they do, Eslint will warn them about it.

Long story short, it doesn't impact the code 👍